Dockerfile

```diff
@@ -62,7 +62,7 @@ RUN mix deps.compile
 COPY . .
 COPY --from=libp2p_builder /libp2p_port/libp2p_port /consensus/priv/native/libp2p_port
 
-RUN protoc --elixir_out=./lib proto/libp2p.proto
```

Comment:
are you sure about this change? in the ci it's `protoc --elixir_out=./lib proto/libp2p.proto`

Reply:
Locally we build with the makefile that uses `.` only. We should update the CI too, just to have them all match.